dbt/include/spark/macros/materializations/table.sql

```diff
@@ -68,6 +74,8 @@ if isinstance(df, pyspark.sql.dataframe.DataFrame):
   pass  # since it is already a Spark DataFrame
 elif pyspark_available and isinstance(df, pyspark.pandas.frame.DataFrame):
   df = df.to_spark()
+elif koalas_available and isinstance(df, databricks.koalas.frame.DataFrame):
+  df = df.to_spark()
```

Comment:
It's also better to convert pandas DF to Koalas DF first similar to `pyspark.pandas`.

Around Line 70:

```py
if pyspark_available and pandas_available and isinstance(df, pandas.core.frame.DataFrame):
  df = pyspark.pandas.frame.DataFrame(df)
elif koalas_available and pandas_available and isinstance(df, pandas.core.frame.DataFrame):
  df = databricks.koalas.frame.DataFrame(df)
```

Reply:
Excellent, thank you making this as efficient as possible! 🤩